packages/tools/viewer/src/viewer.ts

```diff
@@ -280,6 +280,16 @@ export type ViewerOptions = Partial<{
      * Enabled by default.
      */
     autoSuspendRendering: boolean;
+
+    /**
+     * Automatically rotates a 3D model or scene without requiring user interaction. 
+     */
+    cameraAutoOrbit?: boolean;

```

Comment:
I have a post 8.0 release change planned that will make most or all of the properties configurable via options. This would mean having cameraAutoOrbit be the full struct (to avoid a breaking change later). I think if we can do that as part of this PR, it should be reasonable safe without worrying about behavior breaking changes that will come with my bigger change.

```suggestion
    cameraAutoOrbit?: Partial<CameraAutoOrbit>;
```